Client.md

```diff
@@ -1,14 +1,14 @@
 # Overview
 
-All about the Client Schema
+All about the Client schema.
```

Comment:
Little confusing because it is not its own schema anymore it is under NEFSC_GARFO.